.github/workflows/ci.yml

```diff
@@ -123,3 +123,24 @@ jobs:
       run: make -s -j4
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
+
+  out-of-source-and-install:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: extra preparations
+      run: |
+        mkdir ./build
+        mkdir ./install
+    - name: config
+      run: ../config --strict-warnings --prefix=$(cd ../install; pwd) && perl configdata.pm --dump
```

Comment:
should that be `/bin/pwd` to avoid possible shell builtins that might be wrong?  perhaps doesn't matter.

Reply:
`pwd` is there since at least UNIX v7 (I've a copy of the UNIX Programmer's Manual from 1983), I would be surprised if there's a shell that "might be wrong" more than `/bin/pwd` has bugs...  I strongly doubt it matters.